test/test-pkg-config.rb

```diff
@@ -314,4 +314,34 @@ def test_equals_to
                    parse_requires("fribidi = 1.0"))
     end
   end
+
+  sub_test_case("#merge_back_cflags") do
+    def merge_back_cflags(cflags)
+      @glib.__send__(:merge_back_cflags, cflags)
+    end
+
+    def test_d_flag
```

Comment:
`flag` is redundant because this test case is for `merge_back_cflags`.

```suggestion
    def test_d
```